### ISSUE TITLE
How to use in DMA?

### ISSUE THREAD

**Issue by Cyk-Fad** [fastmatikqcc/CR3_Shuffle#1]
As title says,could u plz give some advice?

**Comment by fastmatikqcc** [comment]
> As title says,could u plz give some advice?

Unfortunately I have no experience with dma !

**Comment by Cyk-Fad** [comment]
> > As title says,could u plz give some advice?
> 
> Unfortunately I have no experience with dma !

oh maybe DISCORD or anyelse?

**Comment by fastmatikqcc** [comment]
> > > As title says,could u plz give some advice?
> > 
> > 
> > Unfortunately I have no experience with dma !
> 
> oh maybe DISCORD or anyelse?

Yes I have discord but I can't help myself because I don't know anything about dma
discord : kraft_cheese.

**Comment by Cyk-Fad** [comment]
> > > As title says,could u plz give some advice?
> > 
> > 
> > Unfortunately I have no experience with dma !
> 
> oh maybe DISCORD or anyelse?

HAHA Seemed like many people asked u for DMA fix, I fixed it on DMA. My dc is->ap15_2024
